staging/src/k8s.io/kubectl/pkg/cmd/annotate/annotate_test.go

```diff
@@ -236,6 +236,19 @@ func TestUpdateAnnotations(t *testing.T) {
 				},
 			},
 			annotations: map[string]string{"a": "b"},
+			expected: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{"a": "b"},
+				},
+			},
+		},
+		{
+			obj: &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{"a": "b"},
+				},
+			},
+			annotations: map[string]string{"a": "c"},
 			expectErr:   true,
```

Comment:
Would you be willing to make the same improvement to these tests that you made in the labels PR, so they prove we're getting the specific error we expect? https://github.com/kubernetes/kubernetes/pull/105936#discussion_r740361045

Reply:
No problem. It's done.